wgsl/index.bs

```diff
@@ -5001,27 +5012,41 @@ parameters and return types:
   </xmp>
 </div>
 
-## Function calls TODO ## {#function-calls}
-
-A function call is a statement or expression which invokes a function.
-
-TODO: explain how invocation works: supply operands matching formal parameter types,
-"suspend" execution of the caller, then resume after
-the callee is done (unless discard). Describe return value.
-
-TODO: A function call site is a dynamic context. This matters when discussing the originating
-variable for the dynamic value provided as the operand for a formal argument having pointer type.
-
-The names in the parameter list of a function definition are available for use in the body
-of the function.
-During a particular function evaluation,
-the parameter names denote the values specified to the function call expression or statement
-which initiated the function evaluation;
-the names and values are associated by position.
-
-TODO: define 'formal parameter'.
-
-## Function calls TODO ## {#func-call-semantics}
+## Function calls ## {#function-calls}
+
+A <dfn noexport>function call</dfn> is a statement or expression which invokes a function.
+
+When a function call is executed the following steps occur:
+* Execution of the current function is suspended.
+    All [=function scope=] variables and constants maintain their current values.
+* Storage is allocated to the function scope variable in the called function.
```

Comment:
There's a missing word or something here. Suggest:

"Storage is allocated for each function scope variable in the called function."

Reply:
Maybe this could use a note about initialization, as a reminder.

"Initialization occurs as described in [[#variables]]."